### ISSUE TITLE
Masterscope: `. ANALYZE filename` fails silently

### ISSUE THREAD

**Issue by hjellinek** [Interlisp/medley#1858]
The Masterscope command

```lisp
. ANALYZE filename
```

fails silently.

The corrected command

```lisp
. ANALYZE ON filename
```

works as expected.

**Comment by rmkaplan** [comment]
Perhaps
   . ANALYZE filename 
should at least print a message “filename is not a function” ?

**Comment by MattHeffron** [comment]
This looks like a job for DWIM!

**Comment by rmkaplan** [comment]
No, what if you have a function and a file with the same name?  The syntax for functions and files is different—files require ON.

If you say . ANALYZE ON XXX, you get a file-not-found error.  By symmetry you should get a meaningful message or error if you say . ANALYZE XXX, where XXX is not a function.
